src/pages/components/search/accessibility.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: Search
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListInput,
+  StructuredListCell,
+  OrderedList,
+  ListItem,
+} from 'carbon-components-react';
+
+<PageDescription>
+
+The Search Carbon Component contains a collection of items and objects that,
```

Comment:
```suggestion
The search Carbon component contains a collection of items and objects that,
```

Reply:
I think, since our components are all capitalized that `Search` should stay capitalized but `component` definitely should be lowercase!

Reply:
We made the style decision that components aren't proper nouns and shouldn't be capitalized. All content that is edited or produced is following this policy. Even though the component is capitalized in code, if it's not being used in a code snipped it should be lowercase.